chart/Chart.yaml

```diff
@@ -8,8 +8,8 @@ maintainers:
   - name: Flanksource
     url: https://www.flanksource.com
 icon: https://avatars.githubusercontent.com/u/58787470
-dependencies:
```

Comment:
Why do we comment this out ?

Reply:
THat was supposed to just be temporary while I templated the chart.  Have fixed